datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -155,7 +157,11 @@ public FlowIdentifier triggerDatalakeSyncComponentVersionsFromCmFlow(SdxCluster
     private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
         boolean retVal = true;
         String reason = null;
-        if (cluster.isRangerRazEnabled()) {
+        DetailedEnvironmentResponse environmentResponse = environmentClientService.getByName(cluster.getEnvName());
```

Comment:
Can we check the raz enabled value before doing a database lookup?

Reply:
or just in general moved lower on this check.

Reply:
RAZ check is a temporary check. This will be removed after updating the docs with the additional step to add the ranger policy.

Reply:
will address